storage/innobase/xtrabackup/doc/source/installation/docker.rst

```diff
@@ -62,15 +123,58 @@ intend to use with Percona XtraBackup.
       - `Running in a Docker container
 	<https://www.percona.com/doc/percona-server/LATEST/installation/docker.html>`_
 
-.. code-block:: bash
-
-   $ sudo docker run -d --name percona-server-mysql \
-   -e MYSQL_ROOT_PASSWORD=root percona/percona-server:8.0
 
 As soon as Percona Server for MySQL runs, add some data to it. Now, you are
 ready to make backups with Percona XtraBackup.
 
-Creating a `Docker` container from Percona XtraBackup image
+Use Docker Inspect to find container information 
+==================================================
+
+
+To do a backup, you must find the local directory on the host and the internal IP address that the container is using, so that the xtrabackup container can use it. 
```

Comment:
remove the comma after using

the container is using so that

Reply:
Rewritten